rasa_nlu/config.py

```diff
@@ -70,7 +75,7 @@ def __init__(self, filename=None, env_vars=None, cmdline_args=None):
                 warnings.warn("No pipeline specified and unknown pipeline template " +
                               "'{}' passed.".format(self.__dict__['pipeline']))
 
-        for key, value in self.items():
+        for key, value in list(self.items()):
```

Comment:
in this specific case it's unnecessary true. Was part of the automatic conversion that just ensured exactly uniform return types. Similar for some of the other things you highlighted. Slipped my mind to check some of the automatically converted parts in detail.